Bottom: c6d3371aa54fcf7d526cc4ab9cb007e5dbded790
Top:    e16b6dec36e43ba2217ba4904527865e38e71926
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-14 16:24:53 -0500

update worker configure like master


---
diff --git a/docker/buildbot-master/buildbot/master.cfg b/docker/buildbot-master/buildbot/master.cfg
index 6232036e..c18ebba4 100644
--- a/docker/buildbot-master/buildbot/master.cfg
+++ b/docker/buildbot-master/buildbot/master.cfg
@@ -146,10 +146,10 @@ c['buildbotNetUsageData'] = 'basic'
 
 import openembedded
 reload(openembedded)
-c['workers'].extend(openembedded.workers)
-c['schedulers'].extend(openembedded.schedulers)
-c['builders'].extend(openembedded.builders)
-c['change_source'].extend(openembedded.change_source)
+c['workers'].extend(openembedded.WorkerConfig['workers'])
+c['schedulers'].extend(openembedded.WorkerConfig['schedulers'])
+c['builders'].extend(openembedded.WorkerConfig['builders'])
+c['change_source'].extend(openembedded.WorkerConfig['change_source'])
 
 import ptxdist
 reload(ptxdist)
diff --git a/docker/buildbot-master/buildbot/openembedded.py b/docker/buildbot-master/buildbot/openembedded.py
index ca79dca1..4a67093b 100755
--- a/docker/buildbot-master/buildbot/openembedded.py
+++ b/docker/buildbot-master/buildbot/openembedded.py
@@ -38,6 +38,15 @@ c['workers'] = [
 
 DEFAULT_REPO = 'git@git.novatech-llc.com:ntel/setup-scripts.git'
 
+c = WorkerConfig = {}
+
+
+# BUILDERS
+# The 'builders' list defines the Builders, which tell Buildbot how to perform
+# a build: what steps, and which workers can execute them.  Note that any
+# particular build will only take place on one worker.
+c['builders'] = []
+
 # CHANGESOURCES
 c['change_source'] = [
     changes.GitPoller(
